### DISCUSSION TITLE
Migrate from Piler 1.3.9 with Sphinx scheduled indexing to 1.4.5 with Manticore real-time indexing.

### DISCUSSION THREAD

**Discussion by gkwurst** [jsuto/piler#74] [Q&A] [answered]
Looking at the migration instructions:

https://www.mailpiler.org/migrating-piler-to-a-new-node/

Is there anything special that needs to be done here for this scenario?  Looks like config files moved from /usr/local/etc/piler to /etc/piler.  Move /var/piler/sphinx/* fron old server to /var/piler/manticore/ on new?  Anything needed to get Manticore to reindex the old data?  

Thanks!

**Comment by jsuto** [comment]
You are saying migration, but you describe an upgrade. So which is it? Or perhaps a migration + upgrade? Do you want to move he archive to a new host?

**Reply by gkwurst** [reply]
It's a migration between separate hosts.  I have the new server and Piler up and running and want to move the data over from the old one.

**Reply by jsuto** [reply, marked as answer]
OK. In that case some steps needs tweaking in the migration guide you mentioned.

After you copied data (except sphinx index data) from OLD to NEW host, you need to update the database schema.
Then you need to reindex all emails to populate the manticore search index.

Btw. the configs are where you put them. It depends on the --sysconf-dir configure parameter.

**Reply by gkwurst** [reply]
Thanks, do I just need to run db-upgrade.sql from the 1.4.5 version or do I need to do all the ones between 1.3.9 and 1.4.5?

**Reply by jsuto** [reply]
The db-upgrade.sql in 1.4.5 contains all the relevant changes that you need.

**Reply by gkwurst** [reply]
Perfect.  One last question if you don't mind:  While running the reindex is it ok for the Piler to be receving new mail over smtp, or should I let the reindex finish before opening up the system to new mail?

**Reply by jsuto** [reply]
It's OK to receive emails while reindexing.
